test/source/tests/compose.ts

```diff
@@ -2283,6 +2283,22 @@ export const defineComposeTests = (testVariant: TestVariant, testWithBrowser: Te
       })
     );
 
+    test(
+      'compose - send message with attachments which contain emoji in filename',
+      testWithBrowser(async (t, browser) => {
+        const acctEmail = 'flowcrypt.compatibility@gmail.com';
+        await BrowserRecipe.setupCommonAcctWithAttester(t, browser, 'compatibility', {
+          attester: { includeHumanKey: true },
+        });
+        const subject = `Test Sending Message With Attachment Which Contains Emoji in Filename ${Util.lousyRandom()}`;
+        const composePage = await ComposePageRecipe.openStandalone(t, browser, acctEmail);
+        await ComposePageRecipe.fillMsg(composePage, { to: 'human@flowcrypt.com' }, subject);
+        const fileInput = (await composePage.target.$('input[type=file]')) as ElementHandle<HTMLInputElement>;
+        await fileInput!.uploadFile('test/samples/attac👍hment!🔸.txt');
+        await ComposePageRecipe.sendAndClose(composePage);
```

Comment:
I manually tested using the `attac👍hment!🔸.txt.`.
Sending the email went smoothly, but on the receiver's end, both the filename and file content were incorrect. (file content seems malformed)

Plus, I think it's better to test if sent attachment file name & content is correct in ui test.

![image](https://github.com/FlowCrypt/flowcrypt-browser/assets/77344191/a046400a-896a-4fd0-a6f9-5000bf8dce7c)

Reply:
Hmm, same for me, I think earlier it was sent with the correct attachment name, will check

Reply:
It works in Firefox, but sends incorrect filename in Chrome, will fix it

Reply:
@ioanmo226 I fixed this issue and also updated test to check attachment filename of the sent message